packages/graphback-core/src/annotations/DefaultMetadata.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * An interface that represents the parsed metadata from the `@default(value: <metadata-value>)` annotation.
+ *
+ */
+export interface DefaultMetadata {
```

Comment:
```suggestion
export interface DefaultMarkerMetadata {
```

Reply:
> DefaultMarkerMetadata

Why?

Reply:
`DefaultMetadata` got me confused as `This is default metadata` instead of `DefaultMarkerMetadata`

Reply:
Also we have `annotations` here
We use annotations, markers and directives in many places. 
We need to stick to single name :D

Reply:
How about `DefaultAnnotationMetadata` ?

Reply:
> DefaultValueAnnoation
> DefaultValueMetadata

Reply:
I prefer `DefaultValueAnnotation` - if we are going to use this term in docs we should do the same in code.

Reply:
Anything will work - current name is too short and confusing

Reply:
`DefaultValueAnnotation` is much better.

Reply:
done.